src/Sylius/Bundle/ApiBundle/CommandHandler/Cart/RemoveItemFromCartHandler.php

```diff
@@ -53,6 +58,8 @@ public function __invoke(RemoveItemFromCart $removeItemFromCart): OrderInterface
 
         $this->orderModifier->removeFromOrder($cart, $orderItem);
 
+        $this->orderProcessor->process($cart);
```

Comment:
If I see correctly, order processing is called in `removeFromOrder` method, in `OrderModifier` service, so is it necessary to add order processor here?